### ISSUE TITLE
Bold and italics rendered incorrectly

### ISSUE THREAD

**Issue by geekgonecrazy** [RocketChat/Rocket.Chat.ReactNative#174]
- Your Rocket.Chat.Android app version: latest commit

- Your Rocket.Chat server version: 0.69.3

- Device model (or emulator) you're running with: Pixel

<!-- e.g. For iOS : iPhone 6 - iOS 11.2

- Steps to reproduce:
Put bold and italics on web or any other client.  Then look on this app to see it rendered wrong
![screenshot_20180106-231755](https://user-images.githubusercontent.com/51996/34646846-586fdbba-f338-11e7-8e16-0253e191b2c1.png)
![screenshot_20180106-231759](https://user-images.githubusercontent.com/51996/34646847-5de40dbe-f338-11e7-91e9-7cea5dcd3eaf.png)

**Comment by diegolmello** [comment]
@jgtoriginal Yeah, we could change our fork to do that. I'll do that after I finish my current task.

**Comment by jgtoriginal** [comment]
@diegolmello I just submitted a PR to that repo updating it with the main packages latests updates.
Let me know if you need a hand on this then. Thanks!

**Comment by diegolmello** [comment]
@jgtoriginal Awesome! I'll look into it soon. Thanks!
